app/assets/javascripts/discourse/tests/integration/components/welcome-banner-test.gjs

```diff
@@ -47,4 +50,38 @@ module("Integration | Component | WelcomeBanner", function (hooks) {
         "it does not display when the value transformer returns a different value from the site setting"
       );
   });
+
+  test("optional subheader", async function (assert) {
+    sinon.stub(this.router, "currentRouteName").value("discovery.latest");
+    I18n.translations = {
```

Comment:
Looks like a safer way to do test locale translation overrides is changing `I18n._overrides`, and then making sure you set it to the original value after the test:

https://github.com/discourse/discourse/blob/1503ae3029d670501d227073b113fed02dd62e84/app/assets/javascripts/discourse/tests/unit/localization-test.js#L9-L69

Surprised this isn't breaking other tests actually